CHANGELOG.md

```diff
@@ -1,3 +1,119 @@
+# 0.31.0 (2019-09-??)
+
+## Language changes
+
+- Allow anonymous block args and their type restriction `& : T -> U`. ([#8117](https://github.com/crystal-lang/crystal/pull/8117), thanks @asterite)
```

Comment:
Allow specifying block args without a name?

Reply:
nameless block args? I was looking for a descriptive name as a whole

Reply:
The distinctive feature of a nameless block arg is that it is not captured. So I'd refer to this as non-captured block. This term is also used in the language reference.